hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -266,6 +266,23 @@ objects:
         kind: ClusterRole
         name: splunk-forwarder-operator
 
+    - apiVersion: monitoring.coreos.com/v1
+      kind: PrometheusRule
+      metadata:
+        name: splunk-forwarder-component-unhealthy
+        namespace: openshift-security
+      spec:
+        groups:
+          - name: splunk-forwarder
+            rules:
+              - alert: SplunkForwarderComponentUnhealthy
+                annotations:
+                  message: One or more Splunk Forwarder Components are reporting unhealthy. This issue can impact log forwarding to Splunk, leading to log data loss.
+                expr: splunk_forwarder_component_unhealthy > 0
+                for: 15m
+                labels:
+                  severity: warning
+
```

Comment:
Honestly, I see how it makes sense here, but by convention I think we put these into MCC. What do you think?

Reply:
I wasn't sure if there was a pre-defined pattern for SREP to be fair. I've seen some put it in operator repo, some in MCC. Personally, id rather make it here with the operator as MCC is not automatically updated in FedRAMP, not heavily tested in FedRAMP, and has a higher change to impact our environment for such a small change. While many MCC changes are set to skip FedRAMP clusters its not always the case. 

I'm ok with moving it if there is a strong desire/concern but just my two cents. Im flexible.

Reply:
I'm ok either way too. The PR's already made, for me (as a non-approver) that's good enough reason to keep it here and minimize busy-work.